### PR TITLE
feat(ui): tags

### PR DESCRIPTION
**Pull request by bimalgrg519**
Solves https://github.com/bucketeer-io/bucketeer/issues/178

### CODE REVIEW

**Comment by cre8ivejp** [comment]
@bimalgrg519 
Thank you for your PR!

When you filter the search using two tags, it will delete the first tag if you delete the second one. Can you take a look?

#### How to reproduce

1- List the features by accessing http://localhost:8000/development/features?page=1&sort=-createdAt&tagIds=asdf&tagIds=web
2- Delete the tag `web`
3- Confirm the tag `asdf` is deleted

File: ui/web-v2/apps/admin/src/assets/lang/en.json [hunk 1]
```diff
@@ -233,6 +233,8 @@
   "feature.filter.enabled": "Enabled",
   "feature.filter.hasExperiment": "Has experiment",
   "feature.filter.maintainer": "Maintainer",
+  "feature.filter.tags": "Tags",
+  "feature.filter.tags.placeholder": "Please select one or more tags",
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/assets/lang/en.json:237]: nit: `Select one or more tags`

**Comment by cre8ivejp** [comment]
@bimalgrg519 

Can you align all the input fields to the top when the select option menu changes its height?

Screen Shot 2023-01-23 at 19 40 32

Can you change to keep the options open while selecting the tags on the feature flag and the push create pages, also?

**Comment by cre8ivejp** [comment]
@bimalgrg519
Yes, but can you keep the operator `=` aligned to the middle of the input field on the left like it was before?

214026855-4d07c393-105c-44af-b149-934c4b308e25

**Comment by cre8ivejp** [comment]
@bimalgrg519 

Can you take a look at this issue?
When I close the select option menu without clicking on the add button, it doesn't clear the selected options when I reopen the menu.

Screen Shot 2023-01-23 at 23 06 01

**Comment by cre8ivejp** [comment]
@bimalgrg519 

> Can you change to keep the options open while selecting the tags on the feature flag and the push create pages, also?

How about this comment? Are you still working on it?

**Comment by bimalgrg519** [comment]
> 

Sorry, I didn't see that. Working on it now.

**Review by cre8ivejp — APPROVED** [review]
Nice 🎉 
Thank you!

**Pull request merged**